front/lib/auth.ts

```diff
@@ -153,6 +153,58 @@ export class Authenticator {
     });
   }
 
+  /**
+   * Get an Authenticator for the target workspace associated with the specified user.
```

Comment:
Add a comment saying "to be used only in context where you can't get an authenticator object from a secured key (session or API Key).